app/src/main/res/values/styles.xml

```diff
@@ -106,4 +106,13 @@
     <style name="Alert.Button.Negative" parent="Widget.MaterialComponents.Button.TextButton">
         <item name="android:textColor">@color/colorAccent</item>
     </style>
+
+    <style name="Theme.Transparent" parent="Theme.AppCompat">
```

Comment:
I see 'Home Assistant' in the middle of the screen behind the biometric popup, which looks out of place. Changing the parent theme to `Theme.AppCompat.NoActionBar` removes it.

Reply:
OK